corehq/apps/reports_core/static/reports_core/js/base_template_new.js

```diff
@@ -0,0 +1,110 @@
+hqDefine('reports_core/js/base_template_new', function() {
```

Comment:
You tested these changes locally with both UCRs and standard reports, right?

Reply:
Yup. It worked on my reports, but it might be worth pulling and checking out more complex ones if you have them

Reply:
Cool, that sounds like enough due diligence.